### PR TITLE
build(storybook): Add vercel config for `docs-ui`

### PR DESCRIPTION
**Pull request by billyvg**
Add `now.json` to `docs-ui/` and also output built storybook files to `docs-ui/`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -199,7 +199,7 @@
     "dev-ui": "SENTRY_UI_DEV_ONLY=1 SENTRY_WEBPACK_PROXY_PORT=7999 yarn webpack-dev-server",
     "dev-acceptance": "NO_DEV_SERVER=1 NODE_ENV=development yarn webpack --watch",
     "storybook": "SENTRY_UI_HOT_RELOAD='' start-storybook -p 9001 -c .storybook",
-    "storybook-build": "build-storybook -c .storybook -o .storybook-out",
+    "storybook-build": "build-storybook -c .storybook -o docs-ui/.storybook-out --quiet",
```
Comment (billyvg) [inline package.json:202]: This needs to be relative to the project root dir that is configured in Vercel.

**Comment by billyvg** [comment]
This should prevent github messages from vercel bot.

File: .travis/deploy-storybook.sh [hunk 2]
```diff
@@ -25,8 +25,8 @@ BUCKET_DIR_NAME="branches/${BRANCH_PROCESSED}"
 echo "Bucket directory: ${BUCKET_DIR_NAME}"
 
 # Upload the files
-gsutil cp .storybook-out/favicon.ico "gs://${GS_BUCKET_NAME}/favicon.ico"
-gsutil -m rsync -r -d .storybook-out/ "gs://${GS_BUCKET_NAME}/${BUCKET_DIR_NAME}"
+gsutil cp docs-ui/.storybook-out/favicon.ico "gs://${GS_BUCKET_NAME}/favicon.ico"
+gsutil -m rsync -r -d docs-ui/.storybook-out/ "gs://${GS_BUCKET_NAME}/${BUCKET_DIR_NAME}"
```
Comment (EvanPurkhiser) [inline .travis/deploy-storybook.sh:29]: Will we kill this at some point then?

**Review by evanpurkhiser — APPROVED** [review]

File: .travis/deploy-storybook.sh (hunk 2 above)
Comment (billyvg) [reply .travis/deploy-storybook.sh:29]: Yeah after I can confirm this shit even works lol

**Pull request merged**